packages/code-studio/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Would it make more sense to simply include `postcss.config.js` in the `ignorePatterns` in `.eslintrc`? Or add a comment here about how this is only used in dev